packages/app/package.json

```diff
@@ -71,6 +71,13 @@
       "pathRewrite": {
         "^/circleci/api/": "/"
       }
+    },
+    "/catalog/api": {
+      "target": "http://localhost:3003",
```

Comment:
shouldn't this point to `http://localhost:7000`? so that we can run `yarn start` in the `packages/backend` folder? rather than running every different service on each different port?

Reply:
Makes sense. I'll fix it.

Reply:
Sidenote, but we should aim to get rid of these now imo :grin:

Reply:
@Rugvip the proxying?

Reply:
Yeah, at least in it's current form. Would like something were apps don't have to enumerate all backend endpoints they need to proxy to.

Reply:
🤧 GraphQL 🤧

Reply:
Not only, right, because we still need sort of IoC here for plugins to register their proxies vs whatever central point registering them all?